apps/ucp-ui/src/App.tsx

```diff
@@ -8,16 +8,23 @@ import "@fontsource/work-sans/500.css";
 import "@fontsource/work-sans/700.css";
 import { Experimental_CssVarsProvider as CssVarsProvider } from "@mui/material";
 import { muiTheme } from "./theme";
+import {
+  DefaultPermissions,
+  WidgetHostPermissions,
+} from "./shared/constants/roles";
 
 const App: React.FC = () => {
+  const scope = `${Object.values(DefaultPermissions).join(" ")} ${Object.values(WidgetHostPermissions).join(" ")}`;
```

Comment:
How does scope work if you don't have the necessary permissions?

Reply:
You will get the token, but it won't have access to the endpoints that required certain scopes.